# Table of Content

- [Table of Content](#table-of-content)
- [0-General](#0-general)
  - [0-Introduction](#0-introduction)
  - [0-Objective](#0-objective)
  - [0-Analysis](#0-analysis)
  - [0-Data Dictionary](#0-data-dictionary)
  - [0-Acknowledgements](#0-acknowledgements)
- [1-Sample Size Calculation](#1--sample-size-calculation)

# 0-General
[Back to Table of Content](#table-of-content)

# Sample Size Calculation for the SCFA Meditation RCT

## Introduction
[Back to Table of Content](#table-of-content)

This study investigates whether an 8-week self-administered online meditation program influences gut microbiome composition and SCFA profiles in healthy adults. SCFAs (short-chain fatty acids), including butyrate, acetate, and propionate, are microbial metabolites with immunomodulatory properties. They play a role in maintaining intestinal barrier integrity and modulating systemic inflammation. Given that both SCFAs and the gut microbiome are highly sensitive to dietary intake, the study also includes dietary data (via food frequency questionnaire) as a covariate.


## Study Objectives
[Back to Table of Content](#table-of-content)

1. **Primary Objective**: To evaluate whether fecal SCFA concentrations (particularly iso-butyrate) differ between the meditation group and the active control (audiobook) group after 8 weeks (T2), controlling for baseline values (T0).
2. **Secondary Objective**: To assess changes in gut microbial composition and associations with self-reported stress and mindfulness.


## Study Significance
[Back to Table of Content](#table-of-content)

This will be the first randomized controlled trial to investigate the effect of a fully remote meditation program on gut microbial metabolites. By focusing on SCFAs as functional immune-relevant readouts and incorporating dietary intake as a covariate, the study closes a critical gap in behavioral microbiome research.


## Methods - Study Design
[Back to Table of Content](#table-of-content)

The study uses a randomized controlled trial design with two parallel groups:
- **Meditation Group**: Self-guided online meditation sessions over 8 weeks.
- **Audiobook Control Group**: Equivalent time exposure to emotionally neutral audio material.

Stool samples are collected at:
- **T0 (Baseline)**
- **T2 (Post-Intervention)**

Diet is assessed at T0 and T2 using validated food frequency questionnaires.


## Methods - Outcome Measures
[Back to Table of Content](#table-of-content)

- **Primary Outcome**: Fecal SCFA levels (iso-butyrate in particular), measured via LC-MS.
- **Covariates**: Baseline SCFA level and dietary intake (e.g., fiber consumption, plant-based proportion).


## Sample Size Calculation Rationale
[Back to Table of Content](#table-of-content)

We aim to detect a between-group difference in fecal SCFA (iso-butyrate) concentrations at T2, adjusted for baseline values. Because no RCT to date has isolated the effect of meditation on fecal SCFAs, we base our assumptions on Raman et al. (2023), who observed a statistically significant increase in iso-butyrate following a 60-day combined yoga and vegan-diet intervention.


### Reference Study Data (Raman et al., 2023)
- Significant increase in iso-butyrate from T1 to T2 in meditators: padj = 0.019
- Visual inspection of figures suggests increase from ~0.8 to ~1.1 arbitrary units
- No absolute means or SDs were reported
- Approximate effect size estimated at Cohen's d ≈ 0.4

We conservatively adopt **d = 0.4** as a plausible effect size for the between-group difference in iso-butyrate at T2. We simulate data using a linear mixed model with baseline and diet as covariates.


# 1- Sample Size Calculation
[Back to Table of Content](#table-of-content)

In [122]:
# Load required package
if (!require(pwr)) install.packages("pwr")
library(pwr)

### === INPUT SECTION ===

# Estimated mean difference in iso-butyrate (T2 - T1 in meditators)
# Visual estimate based on Figure 6A in Raman et al. (2023)
# T1 mean = 0.020
# T2 mean = 0.026
# T2 - T1 = 0.026 - 0.020 = 0.006
mean_diff <- 0.006

# Pooled SD estimated from boxplot IQRs:
# IQR_T1 ≈ 0.015 -> SD ≈ 0.015/ 1.35 = 0.0111
# IQR_T2 ≈ 0.020 -> SD ≈ 0.020/ 1.35 = 0.0148
# pooled SD = sqrt((0.0111² + 0.0148²) / 2)
pooled_sd <- 0.0131

# Estimate of variance explained by baseline SCFA (assumption)
r_squared <- 0.30   # i.e. 30%

# === EFFECT SIZE CALCULATION ===

# Cohen's d
cohen_d <- mean_diff / pooled_sd
cat("Cohen's d:", round(cohen_d, 3), "\n")

# Convert to Cohen's f
cohen_f_raw <- cohen_d / 2
cat("Unadjusted Cohen's f:", round(cohen_f_raw, 3), "\n")

# Adjust Cohen's f for R² from baseline covariate
cohen_f_ancova <- cohen_f_raw / sqrt(1 - r_squared)
cat("Adjusted Cohen's f (ANCOVA):", round(cohen_f_ancova, 3), "\n")

# === SAMPLE SIZE CALCULATION ===

# Calculate sample size per group without dropout
result <- pwr.anova.test(k = 2, f = cohen_f_ancova, sig.level = 0.05, power = 0.80)
n_raw <- ceiling(result$n)
cat("Required sample size per group (no dropout):", n_raw, "\n")

# === ADJUST FOR DROPOUT ===

dropout_rate <- 0.20  # 20% dropout
n_corrected <- ceiling(n_raw / (1 - dropout_rate))
cat("Required sample size per group (with 20% dropout):", n_corrected, "\n")


Cohen's d: 0.458 
Unadjusted Cohen's f: 0.229 
Adjusted Cohen's f (ANCOVA): 0.274 
Required sample size per group (no dropout): 54 
Required sample size per group (with 20% dropout): 68 


# 2-Conclusion
[Back to Table of Content](#table-of-content)

The objective of this analysis was to estimate the minimum sample size required per group to ensure adequate statistical power for detecting group differences in short-chain fatty acid (SCFA) concentrations—specifically iso-butyrate levels—at follow-up (T2) in an upcoming randomized controlled trial. Since baseline SCFA concentrations (T1) will be included as a covariate in the primary analysis, an ANCOVA model was selected to account for baseline variability and improve statistical power.

The effect size was estimated based on Figure 6A from Raman et al. (2023), which illustrated iso-butyrate levels in meditators before (T1) and after a two-month preparatory phase (T2). The mean difference between T2 and T1 was visually approximated at 0.006 micromolar proportions, and the pooled standard deviation was estimated at 0.0131, yielding a Cohen’s d of 0.458. This corresponds to a Cohen’s f of 0.229, which was then adjusted to 0.274 under the assumption that baseline SCFA levels explain 30% of the variance in follow-up values (R² = 0.30).

Using this adjusted effect size and targeting a conventional power of 80% with a significance level of 0.05, the analysis indicated that a sample size of 54 participants per group would be sufficient, assuming no attrition. To account for an anticipated 20% dropout rate, the final recommended sample size is 68 participants per group, resulting in a total of 136 participants across both groups.

In conclusion, recruiting 136 participants in total (68 per group) will provide the planned RCT with sufficient power to detect clinically meaningful differences in SCFA outcomes between the intervention and control arms, while accounting for baseline measurements and expected attrition.